### ISSUE TITLE
miscellaneous methods are missing

### ISSUE THREAD

**Issue by phanirithvij** [uber/h3-go#44]
miscellaneous methods are not implemented in go v3

```c
double H3_EXPORT(degsToRads)(double degrees);
double H3_EXPORT(radsToDegs)(double radians);
double H3_EXPORT(pointDistRads)(const GeoCoord *a, const GeoCoord *b);
double H3_EXPORT(pointDistKm)(const GeoCoord *a, const GeoCoord *b);
double H3_EXPORT(pointDistM)(const GeoCoord *a, const GeoCoord *b);
double H3_EXPORT(hexAreaKm2)(int res);
double H3_EXPORT(hexAreaM2)(int res);
double H3_EXPORT(cellAreaRads2)(H3Index h);
double H3_EXPORT(cellAreaKm2)(H3Index h);
double H3_EXPORT(cellAreaM2)(H3Index h);
double H3_EXPORT(edgeLengthKm)(int res);
double H3_EXPORT(edgeLengthM)(int res);
double H3_EXPORT(exactEdgeLengthRads)(H3Index edge);
double H3_EXPORT(exactEdgeLengthKm)(H3Index edge);
double H3_EXPORT(exactEdgeLengthM)(H3Index edge);
int64_t H3_EXPORT(numHexagons)(int res);
int H3_EXPORT(res0IndexCount)();
void H3_EXPORT(getRes0Indexes)(H3Index *out);
int H3_EXPORT(pentagonIndexCount)();
void H3_EXPORT(getPentagonIndexes)(int res, H3Index *out);
```

**Comment by phanirithvij** [comment]
will be fixed by #46

**Comment by joegilley** [comment]
fixed in 3.7.1 (thanks @retbrown!)
